### PR TITLE
Mathieu/feat/login page

### PR DESCRIPTION
**Pull request by MathieuRobilliard**

### CODE REVIEW

File: src/LoginPage.tsx [hunk 1]
```diff
@@ -1,11 +1,56 @@
 import * as React from 'react';
+import 'semantic-ui-css/semantic.min.css';
```
Comment (Porquepix) [inline src/LoginPage.tsx]: I don't think you have to include this file here, if it is included in index.tsx.

File: src/LoginPage.tsx [hunk 2]
```diff
@@ -1,11 +1,56 @@
...
+import { Button, Form, Grid, Header, Image, Message, Segment } from 'semantic-ui-react'
 
 class LoginPage extends React.Component {
```
Comment (Porquepix) [inline src/LoginPage.tsx]: The Login page can be a Stateless component.

File: src/LoginPage.tsx [hunk 3]
```diff
@@ -1,11 +1,56 @@
...
     render() {
       return (
-        <div >
-            LoginPage
-        </div>
+        <div className='login-form'>
+                {}
```
Comment (Porquepix) [inline src/LoginPage.tsx]: ??

File: src/LoginPage.tsx [hunk 4]
```diff
@@ -1,11 +1,56 @@
...
+                <style>{`
```
Comment (Porquepix) [inline src/LoginPage.tsx]: Do you really need this style tag ?
If yes, please put in in its own css file.

File: src/LoginPage.tsx [hunk 5]
```diff
@@ -1,11 +1,56 @@
...
+                body > div,
+                body > div > div,
+                body > div > div > div.login-form {
+                    height: 100%;
+                }
+                `}</style>
+                <Grid
+                textAlign='center'
+                style={{ height: '100%' }}
```
Comment (Porquepix) [inline src/LoginPage.tsx]: Please, avoid to use `style` property. Use class and css instead.

File: src/LoginPage.tsx [hunk 6]
```diff
@@ -1,11 +1,56 @@
...
+                verticalAlign='middle'
+                >
+                <Grid.Column style={{ maxWidth: 450 }}>
+                    <Header as='h2' color='blue' textAlign='center'>
+                    <Image src='/logo.png' />
+                    {' '}Log-in to your Prello account
```
Comment (Porquepix) [inline src/LoginPage.tsx]: ??

File: src/LoginPage.tsx [hunk 7]
```diff
@@ -1,11 +1,56 @@
...
+                    </Header>
+                    <Form size='large'>
```
Comment (Porquepix) [inline src/LoginPage.tsx]: I thnik we can create a LoginForm component.

File: src/LoginPage.tsx [hunk 8]
```diff
@@ -1,11 +1,56 @@
...
+                        icon='lock'
+                        iconPosition='left'
+                        placeholder='Password'
+                        type='password'
+                        />
+                        <Button color='blue' fluid size='large'>Login</Button>
+                    </Segment>
+                    </Form>
+                    <Segment >
+                        <Button color='teal' fluid size='large'>Connect with GitHub</Button>
```
Comment (Porquepix) [inline src/LoginPage.tsx]: I think we can create a GithubLoginButton component.

File: src/LoginPage.tsx (hunk 1 above)
Comment (MathieuRobilliard) [reply src/LoginPage.tsx]: I leave it at a comment, in case of. I'll delete the line later.

**Pull request merged**